workshops/3d_models_with_zdog/README.md

```diff
@@ -0,0 +1,257 @@
+![Render image](https://cloud-k11ck8g2n.vercel.app/0image.png)
+
+Congrats, you just made your first 3d model… Yeah, maybe not what you were expecting. Let’s fix this by animating it.
+
+![woah gif](https://cloud-kr2lyxjbx.vercel.app/0woah.gif)
+
+### Animating it
+Add the following code to our JS file:
+
+```javascript
+    function animatemodel() {
+      ws.rotate.y += 0.01;
+      ws.rotate.x += 0.01;
+      ws.updateRenderGraph()
+      requestAnimationFrame(animatemodel)
+    }
+
+    animatemodel()
+```
+Explanation:
+
+1. We just created a function that will make the 3d model rotate, you can name this function to whatever you want.
+2. ```rotate.x``` and ```rotate.y``` set’s the velocity to the rotation of the model: 
+- The cube will move up and down depending on the given value (- or +, respectively)
+- The cube will move to the right and to the left depeding on the given value (- or +, respectively)
+3. ```ws.updateRenderGraph()``` updates and render your Zdog illustration that was declared in the first variable, make sure to write the correct name in.
+4. ```requestAnimationFrame(animatemodel)``` this is like a loop, basically it makes the model rotates every time by creating frames.
+5. ```animeatemodel()``` calls the function.
+
+Now you can click on run again!
+
+![Animated model](https://cloud-djkuut7y4.vercel.app/0screen_recording_2020-10-03_at_7.31.47_pm.gif)
+
+Amazing, huh?
+
+![Amazing gif](https://cloud-hrs0t8jeh.vercel.app/0tenor.gif)
+
+### Multiple Shapes
+If you want to try and make more complex models you will need to use multiple shapes, [here](https://zzz.dog/#made-with-zdog) are some examples of what you can create:
+
+![examples](https://cloud-2jaw6a14x.vercel.app/0image.png)
+
+Making multiple shapes is very easy, it’s as simple as putting multiple shapes together, until you get what you want. The hardest part of this, is to put everything where it must be, we can do this using the property: “translate”. Let’s see how it works!
+
+I wanna try making the Hack Club logo, but you can make whatever you want!. Click [here](https://repl.it/@wollygfx/Hack-Club-logo) to see the final code.
+
+I'll start making the red square in the background:
+```javascript
+new Zdog.Box({
+  addTo: model,
+  width: 100,
+  height: 100,
+  color: '#ec3750',
+  stroke:20,
+  translate: { z: -18 },
+})
+```
+1. As seen before, the ```addTo``` property puts the made shape inside of the main Zdog Illustration.
+2. Used the width and height properties to make a perfect square, i didn't give it a depth because it was not needed.
+3. The whole square will have the same color (red), so we can use the color property to give the whole shape a single color instead of assigning a color to each face.
+4. As said before, the stroke property helps the shape to look a little rounded.
+5. The translate property moves the square -18 within the z-axis. You can interpret this as if you were giving a shape a depth, so the shape is moved backwards.
+
```

Comment:
```suggestion
1. As seen before, the ```addTo``` property puts the made shape inside of the main Zdog Illustration.
2. Used the width and height properties to make a perfect square, I didn't give it depth because it was not needed.
3. The whole square will have the same color (red), so we can use the color property to give the whole shape a single color instead of assigning a color to each face.
4. As said before, the stroke property helps the shape to look a little rounded.
5. The translate property moves the square -18 within the z-axis. You can interpret this as if you were giving a shape a depth, so the shape is moved backward.

```